radicle-crdt/src/ord.rs

```diff
@@ -5,9 +5,15 @@ use serde::{Deserialize, Serialize};
 
 use crate::Semilattice;
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 #[serde(transparent)]
-pub struct Max<T>(T);
+pub struct Max<T>(pub T);
+
+impl<T: core::fmt::Debug> core::fmt::Debug for Max<T> {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
+        write!(f, r#"Max({:?})"#, self.0)
+    }
+}
```

Comment:
Is this different to what gets generated? :thinking: Is it the quotation marks?

Reply:
Ah, I see. Thanks :)

Reply:
It might be worth noting that, otherwise I imagine someone will come along and wonder the same thing :smile:

Reply:
Sure, will do